rss.py

```diff
@@ -17,27 +18,32 @@ def setup(self, *args, **kwargs):
 
         for feed in self.feeds:
             if feed not in self.storage:
-                self.storage[feed] = ''
+                self.storage[feed] = []
         self.loop_task(self.update_interval, self.check_feeds, now=False)
 
+    def hash(self, entry):
+        return hashlib.sha224("{}{}{}".format(entry.id,
```

Comment:
Wasn't the original problem with entries that had different IDs while their content was the same? As far as I can see, including the entry ID would thus not solve the original problem, or am I mistaken?

Also, why sha224? :-D